src/test/java/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,50 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+
+    @DisplayName("splitAndSum 메소드는 null 또는 빈문자가 입력 되었을때, 결과값은 '0'이 나와야 한다.")
+    @Test
+    public void splitAndSum_메소드_입력값_검증_테스트() {
+        int expected = StringAddCalculator.splitAndSum(null);
+        assertThat(expected).isEqualTo(0);
+
+        expected = StringAddCalculator.splitAndSum("");
+        assertThat(expected).isEqualTo(0);
+    }
+
+    @DisplayName("splitAndSum 메소드는 숫자 문자 하나가 입력 되었을때, 결과값은 Integer로 변경 후 리턴해야 한다.")
+    @Test
+    public void splitAndSum_메소드_입력값_검증_테스트_2() {
+        int expected = StringAddCalculator.splitAndSum("1");
+        assertThat(expected).isEqualTo(1);
+    }
+
+    @DisplayName("splitAndSum 메소드는 문자열이 ', or :' 로 구분되어 입력 되었을때, 숫자의 합을 반환해야 한다.")
```

Comment:
주석이 너무 구체적이면 차후 기획변경이 있을 경우 항상 같이 동기화를 해줘야하는데, 주석은 컴파일에 영향을 받지 않기에 놓치면서 테스트의 실제 동작과 주석이 달라지게될 수 있습니다. 

```suggestion
    @DisplayName("splitAndSum 메소드는 문자열이 유효한 경우, 숫자의 합을 반환한다.)
```

이 정도로만 작성되도 충분하지 않을까 싶네요!

Reply:
네! 감사합니다!